tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -86,10 +86,10 @@ public function reformatsHtml()
             '<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>' .
             '<body></body>' .
             '</html>';
-        $formattedHtml = "<!DOCTYPE HTML>\n" .
-            "<html>\n" .
-            '<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>' . "\n" .
-            "<body></body>\n" .
+        $formattedHtml = "<!DOCTYPE html>\n" .
+            "<html>" .
+            '<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>' .
+            "<body></body>" .
```

Comment:
masterminds/html5 doesn't support reformatting (`$dom->formatOutput`) so that's why this has changed